teamwork/templates/projects/add_resource.html

```diff
@@ -2,14 +2,20 @@
 {% load static %}
 {% load i18n %}
 
+{% block crumb %}
```

Comment:
Same as others: Breadcrumb should be: "Home > CourseName > ProjectName > Add Resource (active)"